js/views/components/moderators/ModeratorCard.js

```diff
@@ -1,13 +1,13 @@
 import _ from 'underscore';
-import BaseVw from '../baseVw';
-import loadTemplate from '../../utils/loadTemplate';
-import app from '../../app';
-import Profile from '../../models/profile/Profile';
-import VerifiedMod, { getModeratorOptions } from './VerifiedMod';
-import { handleLinks } from '../../utils/dom';
-import { launchModeratorDetailsModal } from '../../utils/modalManager';
-import { anySupportedByWallet } from '../../data/walletCurrencies';
-import { getLangByCode } from '../../data/languages';
+import BaseVw from '../../baseVw';
```

Comment:
@jjeffryes ah, not sure if you notice my note in the web repo, but it's configured so we could use absolute paths, which would avoid all these path changing shenanigans when you move a file.